### ISSUE TITLE
Provide binary package from `master`

### ISSUE THREAD

**Issue by firewave** [include-what-you-use/include-what-you-use#1599]
As the CI is producing the binaries anyways and to ease with testing of the current development version I am wondering if it would be possible to provide a (portable) package.

**Comment by kimgr** [comment]
I decided quite a long while back that I don't want to be responsible for binary releases. See e.g.

* https://github.com/include-what-you-use/include-what-you-use/issues/798
* https://github.com/include-what-you-use/include-what-you-use/issues/581

This is simply not support work I have time and energy for.

**Comment by firewave** [comment]
Fair enough. But since the job already generates a binary that could simply be stored as an artifact which would just be an additional step in the workflow. I could add that.

**Comment by kimgr** [comment]
The work is not pushing the binary out; the work is supporting people asking why that best-effort binary doesn't run on their platform.

**Comment by firewave** [comment]
I did not say that it should be advertised. 😃

But nobody but me has asking for it and I am able to build it myself now - so problem solved. My problem now is somehow getting older versions - but that is out of scope.
